app/scripts/directives/resources.js

```diff
@@ -24,10 +71,18 @@ angular.module('openshiftConsole')
         imagesByDockerReference: '=',
         builds: '=',
         detailed: '=?',
+        initContainers: '=?',
```

Comment:
Same comment here. I'd remove `initContainers`, have one `pod-template` directive, and just show init containers when `detailed` is enabled. This means you'll need to pull the contents from the `ng-repeat` in the view for a container into a separate directive, maybe `pod-template-container`.